### ISSUE TITLE
Improve axe core test element targeting to avoid testing admin bar elements

### ISSUE THREAD

**Issue by skullzarmy** [skullzarmy/a11y-tester-wordpress-plugin#9]
Currently axe core is running on the entire iframe, which contains an admin session, thus an admin bar. The admin bar is not fully a11y compliant, thus appears in the violations report.
